src/app/map/map.component.html

```diff
@@ -352,21 +352,26 @@
                             <br />
                             <label
                                 class="filter-label"
+                                aria-label="Select an option"
```

Comment:
We might want to only have 1 aria-label. I'm not sure how a screen reader would interpret it.